_posts/2020-10-02-snowplow-javascript-tracker-2-16-0-released/index.md

```diff
@@ -16,32 +16,42 @@ We are pleased to announce a new release of the [Snowplow JavaScript Tracker][js
 Read on below for:
 
 1. [Switching build tooling](#1-switching-build-tooling)
-2. [Other features and bug fixes](#2-other-features-and-bugfixes)
-3. [Upgrading](#3-upgrading)
-4. [Documentation and help](#4-documentation-and-help)
+2. [Update Snowplow Micro to v1](#2-update-snowplow-micro-to-v1)
+3. [Other features and bug fixes](#3-other-features-and-bugfixes)
+4. [Upgrading](#4-upgrading)
+5. [Documentation and help](#5-documentation-and-help)
 
 <!--more-->
 
 ## 1. Switching build tooling
 
 As originally suggested by `bernardosrulzon`, we have switched to using the Closure Compiler ([GitHub issue #583](https://github.com/snowplow/snowplow-javascript-tracker/issues/583)). We have also switched to using rollup.js and gulp ([GitHub issue #752](https://github.com/snowplow/snowplow-javascript-tracker/issues/752)). Together, these two updates reduce the `sp.js` filesize by 30%. 
 
-## 2. Other features and bug fixes
+
+## 2. Update Snowplow Micro to v1
+
+We have recently release v1.0.0 of Snowplow Micro, a very small version of a full Snowplow data collection pipeline that it can be launched by a test suite to include tracking in automated tests. You can read more about it in the [release post](https://snowplowanalytics.com/blog/2020/09/11/snowplow-micro-1-0-0-released/). 
```

Comment:
```suggestion
We have recently released v1.0.0 of Snowplow Micro, a very small version of a full Snowplow data collection pipeline that it can be launched by a test suite to include tracking in automated tests. You can read more about it in the [release post](https://snowplowanalytics.com/blog/2020/09/11/snowplow-micro-1-0-0-released/). 
```